lms/static/certificates/sass/views/_validate.scss

```diff
@@ -0,0 +1,288 @@
+// edX: Certificates - View: DIY Validation
```

Comment:
NOTE: styling for self-validation views (currently not included in this feature's development work).